### PR TITLE
HDInsight job view bug fix

### PR DESCRIPTION
**Pull request by voidfunction**

### CODE REVIEW

File: PluginsAndFeatures/AddLibrary/AzureLibraries/com.microsoft.azuretools.sdk/dependencies/ThirdPartyNotices.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ Section A: Microsoft is offering you a license to use the following components w
 35.	Retrofit (https://github.com/square/retrofit)
 36.	RxJava (https://github.com/ReactiveX/RxJava)
 37.	Simple Logging Facade for Java (https://www.slf4j.org/)
-
+38. HtmlUnit (http://htmlunit.sourceforge.net/)
```
Comment (ziyel) [inline PluginsAndFeatures/AddLibrary/AzureLibraries/com.microsoft.azuretools.sdk/dependencies/ThirdPartyNotices.txt]: Do we need to append the open source license of `HtmlUnit` to the end of this document?

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/ActionHttpHandler.java [hunk 2]
```diff
@@ -45,19 +42,14 @@ public void handle(HttpExchange httpExchange) throws IOException {
...
-                    e.printStackTrace();
-                }
-
-
+            try {
+                Application application = JobViewCacheManager.getSingleSparkApplication(new ApplicationKey(requestDetail.getCluster(), requestDetail.getAppId()));
+                JobUtils.openSparkUIHistory(clusterConnectString, requestDetail.getAppId(), application.getLastAttemptId());
+                JobUtils.setResponse(httpExchange, "open browser successfully");
+            } catch (ExecutionException e) {
+                JobUtils.setResponse(httpExchange, "open browser error", 500);
```
Comment (ziyel) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/ActionHttpHandler.java]: Is it possible to add more error message in response? Such as the details of the exception. For now, if user send back the error message, we could not figure out the root cause of browser open failed.

File: PluginsAndFeatures/AddLibrary/AzureLibraries/com.microsoft.azuretools.sdk/dependencies/ThirdPartyNotices.txt (hunk 1 above)
Comment (voidfunction) [inline PluginsAndFeatures/AddLibrary/AzureLibraries/com.microsoft.azuretools.sdk/dependencies/ThirdPartyNotices.txt]: it's Apache 2.0. Already in the document.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/ActionHttpHandler.java (hunk 2 above)
Comment (voidfunction) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/jobs/ActionHttpHandler.java]: Sounds good. Will log the exception information.

**Pull request merged**